sources/lib/Session/Session.php

```diff
@@ -291,6 +291,45 @@ public function getClientUsingPooler($type, $identifier)
     }
 
     /**
+     * Get query manager shortcut
+     *
+     * @param   string                   $identifier
+     * @throws  FoundationException      if no poolers found
+     * @return  \PommProject\Foundation\QueryManager\QueryManagerClient
+     */
+    public function getQueryManager($identifier = null)
+    {
+        return $this->getClientUsingPooler('query_manager', $identifier);
+    }
+
+    /**
+     * Get prepared query manager shortcut
+     *
+     * @param   string $identifier
+     * @throws  FoundationException      if no poolers found
+     * @return  \PommProject\Foundation\PreparedQuery\PreparedQueryManager
+     */
+    public function getPreparedQueryManager($identifier = null)
+    {
+        return $this->getClientUsingPooler('prepared_query', $identifier);
+    }
+
+    /**
+     * Get model shortcut
+     *
+     * Without the pomm-project/model-manage package or if the class name
+     * does not exist, this method raise a FoundationException.
+     *
+     * @param   string                   $className model class name
+     * @throws  FoundationException      if no poolers found
+     * @return  \PommProject\ModelManager\Model\Model
```

Comment:
Note that from here I'm referencing a class outside of this package, and this might cause a dependency problem. For now, because PHP cannot type return types properly and this is only done from within the PHPdoc, there is no real problem.

A better solution might be to add this method on a class extending the session inside the pomm Symfony bundle or from the pomm bridge, what do you think?